### ISSUE TITLE
Loading Chunks Crash

### ISSUE THREAD

**Issue by asciidude** [PaperMC/Paper#3493]
### What behaviour is expected:
For the server not to crash

### What behaviour is observed:
It crashed

### Steps/models to reproduce:
Loading chunks

### Plugin list:
Bedwars, WorldEdit, PermissionsEx, Buycraft, Vault, Spark, WorldGuard, SkyWars, ASkyBlock, Essentials, EssentialsChat, Bounty, EssentialsSpawn

### Paper build number:
4c7641d (1.8.8)

### Anything else:
java.util.concurrent.ExecutionException: net.minecraft.server.v1_8_R3.ReportedException: Exception generating new chunk
        at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_252]
        at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_252]
        at net.minecraft.server.v1_8_R3.SystemUtils.a(SourceFile:45) [paper.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.MinecraftServer.B(MinecraftServer.java:774) [paper.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.DedicatedServer.B(DedicatedServer.java:378) [paper.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.MinecraftServer.A(MinecraftServer.java:713) [paper.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.MinecraftServer.run(MinecraftServer.java:616) [paper.jar:git-PaperSpigot-"4c7641d"]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_252]
Caused by: net.minecraft.server.v1_8_R3.ReportedException: Exception generating new chunk
        at net.minecraft.server.v1_8_R3.ChunkProviderServer.originalGetChunkAt(ChunkProviderServer.java:170) ~[paper.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.ChunkProviderServer.getChunkAt(ChunkProviderServer.java:138) ~[paper.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.PlayerChunkMap$PlayerChunk.<init>(PlayerChunkMap.java:373) ~[paper.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.PlayerChunkMap.a(PlayerChunkMap.java:93) ~[paper.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.PlayerChunkMap.b(PlayerChunkMap.java:187) ~[paper.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.PlayerChunkMap.movePlayer(PlayerChunkMap.java:253) ~[paper.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.PlayerList.d(PlayerList.java:354) ~[paper.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.PlayerConnection.a(PlayerConnection.java:458) ~[paper.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.PacketPlayInFlying.a(SourceFile:126) ~[paper.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.PacketPlayInFlying$PacketPlayInPosition.a(SourceFile:57) ~[paper.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.PlayerConnectionUtils$1.run(SourceFile:13) ~[paper.jar:git-PaperSpigot-"4c7641d"]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_252]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_252]
        at net.minecraft.server.v1_8_R3.SystemUtils.a(SourceFile:44) ~[paper.jar:git-PaperSpigot-"4c7641d"]
        ... 5 more
Caused by: java.lang.OutOfMemoryError: Java heap space
        at net.minecraft.server.v1_8_R3.ChunkSection.<init>(ChunkSection.java:15) ~[paper.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.Chunk.<init>(Chunk.java:178) ~[paper.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.ChunkProviderGenerate.getOrCreateChunk(ChunkProviderGenerate.java:187) ~[paper.jar:git-PaperSpigot-"4c7641d"]
        at org.bukkit.craftbukkit.v1_8_R3.generator.NormalChunkGenerator.getOrCreateChunk(NormalChunkGenerator.java:41) ~[paper.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.ChunkProviderServer.originalGetChunkAt(ChunkProviderServer.java:162) ~[paper.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.ChunkProviderServer.getChunkAt(ChunkProviderServer.java:138) ~[paper.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.PlayerChunkMap$PlayerChunk.<init>(PlayerChunkMap.java:373) ~[paper.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.PlayerChunkMap.a(PlayerChunkMap.java:93) ~[paper.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.PlayerChunkMap.b(PlayerChunkMap.java:187) ~[paper.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.PlayerChunkMap.movePlayer(PlayerChunkMap.java:253) ~[paper.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.PlayerList.d(PlayerList.java:354) ~[paper.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.PlayerConnection.a(PlayerConnection.java:458) ~[paper.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.PacketPlayInFlying.a(SourceFile:126) ~[paper.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.PacketPlayInFlying$PacketPlayInPosition.a(SourceFile:57) ~[paper.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.PlayerConnectionUtils$1.run(SourceFile:13) ~[paper.jar:git-PaperSpigot-"4c7641d"]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_252]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_252]
        at net.minecraft.server.v1_8_R3.SystemUtils.a(SourceFile:44) ~[paper.jar:git-PaperSpigot-"4c7641d"]
        ... 5 more

**Comment by electronicboy** [comment]
1.8.8 is not supported

Scrolling up, the error is in plain English:
> Caused by: java.lang.OutOfMemoryError: Java heap space
